runtime/src/bank.rs

```diff
@@ -1888,6 +1888,121 @@ pub fn goto_end_of_slot(bank: &mut Bank) {
     }
 }
 
+// The `bank` struct as defined in Solana v0.22.3
+#[derive(Default, Deserialize, Serialize)]
+pub struct LegacyBank0223 {
+    #[serde(skip)]
+    pub rc: BankRc,
```

Comment:
todo to me: not bother to check some are missing. I hope compilers and the unit tests do the right job. :)